翻译客服文档 (#833)
taptap/tds-docs: i18n/en/docusaurus-plugin-content-docs/current/sdk/_partials/unity-sdk-installation.mdx

import CodeBlock from "@theme/CodeBlock";
import sdkVersions from "/src/docComponents/sdkVersions";

The SDK can be imported either **through Unity Package Manager** or manually**. Please choose according to your project needs.

#### Method 1: Use Unity Package Manager

##### NPMJS Installation

As of version 3.25.0, TapSDK supports NPMJS installation, with the advantage that only the version number needs to be configured and nested dependencies are supported.

Add the following dependencies to your project's `Packages/manifest.json` file:

<CodeBlock className="json">
  {`"dependencies":{
    ${props.npmDeps.map(dep => `"${dep}":"${sdkVersions.taptap.unity}",`).join('\n    ')}
}`}
</CodeBlock>

> Note, however, that `scopedRegistries` should be declared at the `dependencies` level in `Packages/manifest.json`:

Reviewer: ```suggestion
However, it should be noted that `scopedRegistries` must be declared under the same level of dependencies in `Packages/manifest.json`:
```

<CodeBlock className="json">
{`"scopedRegistries": [
    {
      "name": "NPMJS",
      "url": "https://registry.npmjs.org/",
      "scopes": ["com.tapsdk", "com.taptap", "com.leancloud"]
    }
  ]`}
</CodeBlock>

##### GitHub Installation

Add the following dependencies to your project's `Packages/manifest.json` file:

<CodeBlock className="json">
  {`"dependencies":{
    ${props.githubDeps.map(dep => `"${dep.package}":"${dep.url}",`).join('\n    ')}
}`}
</CodeBlock>

Select **Window > Package Manager** in the top menu of Unity to see the packages already installed in your project.

> #### Method 2: Manual Import

Reviewer: ```suggestion
#### Method 2: Import Manually
```

> 1. Find the **TapSDK Unity** and **LeanCloud C# SDK** downloads on the [download page](/tap-download) and download `TapSDK-UnityPackage.zip` and `LeanCloud-SDK-Realtime-Unity.zip` respectively.

Reviewer: ```suggestion
1. Find the download addresses of **TapSDK Unity** and **LeanCloud C# SDK** on the [download page](/tap-download) , and download `TapSDK-UnityPackage.zip` and `LeanCloud-SDK-Realtime-Unity.zip` respectively.
```

2. In the Unity project, go to **Assets > Import Packages > Custom Packages**, and from the unzipped `TapSDK-UnityPackage.zip`, select the TapSDK packages that you want to use in the game, and import them:

  <ul>
    {props.unitypackageModules.map(module => <li><code>{module.name}</code> {module.desc}。</li>)}
  </ul>

> 3. Unzip the `LeanCloud-SDK-Realtime-Unity.zip` into the Plugins folder and drag and drop it into Unity.

Reviewer: ```suggestion
3. The decompresses `LeanCloud-SDK-Realtime-Unity.zip` is in the Plugins folder, drag and drop it to Unity.
```
